docs/Command-Line-Interface.md

```diff
@@ -20,6 +20,32 @@ Or, if no header is provided, the default hostname and port (set in the env file
 
     /<path>
 
+## Wildcard Input
+
+Most of the commands which require path components allow wildcard arguments for ease of use. For
+example:
+
+```
+bin/tachyon tfs rm /data/2014*
+```
+
+The example command would delete anything in the `data` directory with a prefix of `2014`.
+
+Note that some shells will attempt to glob the input paths, causing strange errors like
+
+```
+rm takes 1 arguments,  not 21
```

Comment:
I was confused by the number 21 and actually spent some time thinking why this number comes here. Could you explain a little bit (e.g., there are 21 other files in local file systems matched?)